code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm

```diff
@@ -174,36 +216,37 @@
 	name = "Sprinkles"
 	id = "sprinkles"
 	description = "Multi-colored little bits of sugar, commonly found on donuts. Loved by cops."
+	taste_description = "childhood whimsy"
 	nutriment_factor = 1
 	color = "#FF00FF"
 
 /datum/reagent/nutriment/mint
 	name = "Mint"
 	id = "mint"
 	description = "Also known as Mentha."
+	taste_description = "mint"
 	reagent_state = LIQUID
 	color = "#CF3600"
 
 /datum/reagent/lipozine // The anti-nutriment.
 	name = "Lipozine"
 	id = "lipozine"
 	description = "A chemical compound that causes a powerful fat-burning reaction."
+	taste_description = "mothballs"
 	reagent_state = LIQUID
 	color = "#BBEDA4"
 	overdose = REAGENTS_OVERDOSE
 
 /datum/reagent/lipozine/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
 	M.nutrition = max(M.nutrition - 10 * removed, 0)
-	M.overeatduration = 0
-	if(M.nutrition < 0)
-		M.nutrition = 0
```

Comment:
these lines would reset overeating. The new way simply drains nutrition and calls it a day since life() probably handles that anyway.